lib/generators/templates/migration.erb

```diff
@@ -1,8 +1,7 @@
-class InlineFormsCreate<%= table_name.camelize %> < ActiveRecord::Migration
+class InlineFormsCreate<%= table_name.camelize %> < ActiveRecord::Migration[5.0]
 
   def self.up
-    #create_table :<%= table_name + ", :id => " + @create_id.to_s %> do |t|
```

Comment:
I think this line is more clear to read. id: (true or false). So let's rewrite that in the next version.